docs/install_guides/installing-openvino-from-archive-macos.md

```diff
@@ -163,6 +163,7 @@ Additional Resources
    To learn more about converting models from specific frameworks, go to:
    * :ref:`Convert Your Caffe Model <convert model caffe>`
    * :ref:`Convert Your TensorFlow Model <convert model tf>`
+   * :ref:`Convert Your TensorFlow Lite Model <convert model tfl>`
```

Comment:
for the install guides and <convert model tfl> symbol, I think it was decided to not show full conversion section here since it's originally placed in the comment and is not displayed